wqflask/tests/wqflask/show_trait/test_export_trait_data.py

```diff
@@ -0,0 +1,103 @@
+import unittest
+from wqflask.show_trait.export_trait_data import dict_to_sorted_list,cmp_samples
+class TestDictionaryList(unittest.TestCase):
+	def setUp(self):
+		self.sample1={
+		"other":"exp1",
+		"name":"exp2"
+		}
+		self.sample2={
+		'se':1,
+		'num_cases':4,
+		'value':6,
+		"name":3
+
+		}
+		self.reversed={
+		"name":3,
+		'value':6,
+		'num_cases':4,
+		'se':1
+		}
+		self.oneItem={
+		'item1':"one"
+		}
```

Comment:
**issue:** Move these variables to where actual variables are. You only stick variables in the `SetUp` method if they will be re-used in different places in the code base.